app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -75,13 +75,21 @@ export const simulateEstimateGas = async (ethereumClientService: EthereumClientS
 	const transactionCount = getSimulatedTransactionCount(ethereumClientService, simulationState, sendAddress)
 	const block = await ethereumClientService.getBlock()
 	const maxGas = simulationGasLeft(simulationState, block)
+
+	const getGasPriceFields = (data: DappRequestTransaction) => {
+		if (data.gasPrice !== undefined) return { maxFeePerGas: data.gasPrice, maxPriorityFeePerGas: data.gasPrice }
```

Comment:
```suggestion
		if (data.gasPrice !== undefined && data.gasPrice !== null) return { maxFeePerGas: data.gasPrice, maxPriorityFeePerGas: data.gasPrice }
```

Reply:
gasprice cannot be null by its current type